extensions/amp-social-share/amp-social-share.md

```diff
@@ -271,6 +271,34 @@ When customizing the style of an `amp-social-share` icon please ensure that the
 
 ## Accessibility
 
+### Indication of focus
+
+The `amp-social-share` element defaults to a blue outline as a visible focus indicator. It also defaults `tabindex=0` making it easy for a user to follow along as he or she tabs through multiple `amp-social-share` elements used together on a page.
+
+The default focus indicator is achieved with the following CSS rule-set.
+
+```css
+amp-social-share:focus {
+  outline: #0389ff solid 2px;
+  outline-offset: 2px;
+}
+```
+
+The default focus indicator can be overwritten by defining CSS styles for focus and including them within a `style` tag on an AMP HTML page. In the example below, the first CSS rule-set removes the focus indicator on all `amp-social-share` elements by setting the `outline` property to `none`. The second rule-set specifies a red outline (instead of the default blue) and also sets the `outline-offset` to be `3px` for all `amp-social-share` elements with the class `custom-focus`.
```

Comment:
Does the component actually support the class `custom-focus`? I'm not seeing this class mentioned elsewhere from the docs in this PR. Let me know if I'm missing something. Otherwise please remove this part and update the code snippet with a working sample.

Reply:
The idea here is that I wanted to provide an example of how the user could overwrite the focus.  The first css ruleset removes the default focus from all `amp-social-share` elements.  The second one (`custom-focus`) applies a new customized focus visual to only the `amp-social-share` elements which include `custom-focus` in their classList.  So the actual `custom-focus` class name isn't that important, it's just the idea of applying the ruleset via a class.

The documentation is matched with the original change from `0.1`: https://github.com/ampproject/amphtml/pull/30775/files

Hope that helps clarify!! :)

Reply:
I've added the following line to the `0.1` and `1.0` docs.  Let me know if you think it's descriptive enough:

```
The user would add the `custom-focus` class to any `amp-social-share` elements he or she wishes to apply this rule-set to.
```

Reply:
I'm wondering if this is overly specific? Surely our devs know how to customize `:focus {}` styling. We just want to let them know that it works as expected.

Reply:
Sounds good, I've removed the last part to make it a bit simpler / general.